core/templates/pages/creator-dashboard-page/creator-dashboard-page.controller.ts

```diff
@@ -273,6 +273,8 @@ angular.module('oppia').directive('creatorDashboardPage', [
               CreatorDashboardBackendApiService.fetchDashboardData());
             dashboardDataPromise.then(
               function(response) {
+                // TODO(#9364): Remove the following line once this file and
```

Comment:
Please update the text of the issue #9364 per the comment I left there, and assign it to yourself for future tracking. Address it at the first opportunity.

Reply:
https://github.com/oppia/oppia/issues/9364#issuecomment-634519460

Reply:
Removed that. Done.